File: Templates/Challenge/template.go
package main

//
// See https://theweeklychallenge.org/blog/perl-weekly-challenge-000
//

//
// Run as: go run ch-1.go < input-file
//

import (
    "fmt"
    "bufio"
    "os"
)

func main () {
    var reader = bufio . NewReader (os. Stdin)
    for {
        var text, err = reader . ReadString ('\n')
        if (err != nil) {
            break
        }
    }
}

File: Templates/Challenge/template.r
#
# See https://theweeklychallenge.org/blog/perl-weekly-challenge-000
#

#
# Run as: Rscript ch-1.r < input-file
#

stdin <- file ('stdin', 'r')
repeat {
    n <- readLines (stdin, n = 1)
    if (length (n) == 0) {
        break
    }
    n = as.integer (n)
}
